src/api/json/catalog.json

```diff
@@ -7704,6 +7704,27 @@
       "versions": {
         "19.0.0": "https://json.schemastore.org/vtcfg-v19.0.0.json"
       }
+    },
+    {
+      "name": "Airbyte Declarative Connectors Specification (manifest.yaml)",
+      "description": "Airbyte Specification for custom connectors",
+      "fileMatch": [
+        "manifest.yaml",
```

Comment:
manifest.yaml is too generic. Can you add a known folder name to the pattern or remove it?